btree/btree.c

```diff
@@ -2995,78 +3032,84 @@ struct dir_rec {
 	uint8_t val_offset;
 };
 
-struct vkv_head {
-	struct m0_format_header  vkv_fmt;      /*< Node Header */
-	struct node_header       vkv_seg;      /*< Node type information */
-	uint16_t                 vkv_used;     /*< Count of records */
-	uint8_t                  vkv_shift;    /*< Node size as pow-of-2 */
-	uint8_t                  vkv_level;    /*< Level in Btree */
-	struct dir_rec          *dir_address;  /*< starting address of dir */
+struct vkvv_head {
+	struct m0_format_header  vkvv_fmt;      /*< Node Header */
+	struct node_header       vkvv_seg;      /*< Node type information */
+
```

Comment:
Remove this blank line since the following comment applies to the above 2 structures.